community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/PatternExpressionAcceptanceTest.scala

```diff
@@ -315,6 +336,7 @@ class PatternExpressionAcceptanceTest extends ExecutionEngineFunSuite with Match
     ))
   }
 
+  // TCK'd
```

Comment:
I don't know if we should mark these tests - ones which include introspecting the plan, as per line 353 - as TCK'ed...that is, the actual test is TCK'ed, but not every aspect as envisaged by this test. I have come across similar. Maybe we ought to discuss...